src/shaders/globe_raster.fragment.glsl

```diff
@@ -1,7 +1,36 @@
 uniform sampler2D u_image0;
 varying vec2 v_pos0;
 
+uniform vec3 u_frustum_tl;
+uniform vec3 u_frustum_tr;
+uniform vec3 u_frustum_br;
+uniform vec3 u_frustum_bl;
+uniform highp vec3 u_globe_pos;
+uniform highp float u_globe_radius;
+uniform bool u_is_antialias;
+uniform vec2 u_viewport;
```

Comment:
These variable may also be used by fog/atmosphere and injected globally to all shaders, to prevent conflict and have the shader compile you can wrap the ones that are shared under a preprocessor define, such as:

```
#ifndef FOG
uniform vec3 u_frustum_tl;
uniform vec3 u_frustum_tr;
uniform vec3 u_frustum_br;
uniform vec3 u_frustum_bl;
uniform highp vec3 u_globe_pos;
uniform highp float u_globe_radius;
uniform vec2 u_viewport;
#endif

uniform bool u_is_antialias;
```